x-pack/plugin/eql/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/eql/10_basic.yml

```diff
@@ -158,3 +158,55 @@ setup:
       catch: missing
       eql.delete:
         id: $id
+
+---
+"EQL status API":
+
+  - do:
+      eql.search:
+        index: eql_test
+        keep_on_completion: true
+        wait_for_completion_timeout: "0ms"
+        body:
+          query: 'process where user == "SYSTEM"'
+  - is_true: id
+  - set: { id: id }
+
+  # status for not completed eql search
+  - do:
+      eql.get_status:
+        id: $id
+  - match: { id: $id }
+  - match: { is_running: true }
+  - match: { is_partial: true }
```

Comment:
These two checks will probably fail occasionally due to a race condition with task completion.

Reply:
@imotov  Thanks! Good  feedback! Indeed,  we are not certain if the task is completed or not by this time!  I changed the test in 4eca9a0b6fe029fc245cd1c7612717fac1f13ba3 to test for less.